server/datastore/mysql/hosts.go

```diff
@@ -2224,9 +2536,7 @@ func (ds *Datastore) UpdateHost(ctx context.Context, host *fleet.Host) error {
 			primary_ip = ?,
 			primary_mac = ?,
 			public_ip = ?,
-			refetch_requested = ?,
-			gigs_disk_space_available = ?,
-			percent_disk_space_available = ?
```

Comment:
So the plan is to stop using these columns and eventually we'll drop them?

Reply:
Correct, the ticket is created to remove them in a future release: https://github.com/fleetdm/fleet/issues/7855